.circleci/template.yml

```diff
@@ -748,14 +752,49 @@ workflows:
           executor: otp_25
           platform: centos_7
           context: mongooseim-org
-          otp_package: "25.0.1-1"
+          otp_package: "25.0.3-1"
+          filters: *all_tags
+      - package:
+          name: rockylinux_8
+          executor: otp_24
+          platform: rockylinux_8
+          context: mongooseim-org
+          otp_package: "24.1.7-1"
+          filters: *all_tags
+      - package:
+          name: almalinux_8
+          executor: otp_24
+          platform: almalinux_8
+          context: mongooseim-org
+          otp_package: "24.1.7-1"
```

Comment:
any reason to use an older version?

Reply:
Currently there is no otp 25 package available for almalinux and rockylinux. I believe that when the new site with packages is running we could expect some newer version published.